### DISCUSSION TITLE
Hello there, how can i fix this failure in routing?

### DISCUSSION THREAD

**Discussion by michelhfmn** [jorisdugue/h5p-bundle#37] [Q&A] [answered]
I get displayed the overview of all H5P's.
I was searching for the bug like 8-9 hours dumping out everything in JS and in the controllers but there is everything fine.

When i click on edit the bundle is searching for the JS and CSS in  /h5p/edit/h5p/libraries/ but it should search in /h5p/libraries
 
 http://127.0.0.1:8000/h5p/edit/h5p/libraries/H5P.MultiChoice-1.14/presave.js?ver=1.14.7

and when i try to create a new one it is searching in /h5p/h5p/libraries

/h5p/h5p/libraries/H5PEditor.RangeList-1.0/h5p-editor-range-list.js?ver=1.0.11

the webroot is in the public folder.

this is the folder structure:

    -public
         -bundles
             -studithh5p
                  -h5p
                       -h5p-core
                       -h5p-editor

         -h5p
             -libraries
                  - all libraries
             -temp

Thank u for your time

**Comment by michelhfmn** [comment]
i thing that is a problem with the server, someone have a tutorial for this: 

First add a virtual host that points to you project. Then in your browser go to http://<your virtualhost>/h5p/list

i tried 2/3 but that doesnt worked

**Comment by jorisdugue** [comment, marked as answer]
Hey,
How is the bundle configured in your project?

if you add `/h5p` in `storage_dir` it will solve the problem 

My current configuration is as follows:

```php
studit_h5_p:
  use_permission: true # This is false by default to let the demo work out of the box.
  storage_dir: "/h5p" # Location to store all H5P libraries and files
  web_dir: "/public" # Location of the public web directory
  export: 3 #for all
  embed: 3 #for all```

**Reply by michelhfmn** [reply]
Thank you very much that worked.

Im on Symfony 5 but your configuration only run with the symfony 4 config, is that a bug?

**Reply by jorisdugue** [reply]
The configuration between symfony 4 and 5 to change due to the change of the architecture. This has created some incompatibility ... I will take care of updating the package.
